entries/jQuery.cssNumber.xml

```diff
@@ -1,31 +1,16 @@
 <?xml version="1.0"?>
-<entry type="property" name="jQuery.cssNumber" return="Object">
+<entry type="property" name="jQuery.cssNumber" return="Object" removed="4.0">
   <title>jQuery.cssNumber</title>
   <signature>
     <added>1.4.3</added>
   </signature>
-  <desc>An object containing all CSS properties that may be used without a unit. The <a href="/css/"><code>.css()</code></a> method uses this object to see if it may append <code>px</code> to unitless values.</desc>
+  <desc>An object containing all CSS properties that may be used without a unit. Prior to jQuery 4.0, the <a href="/css/"><code>.css()</code></a> method uses this object to see if it may append <code>px</code> to unitless values.</desc>
   <longdesc>
-    <p>You can think about <code>jQuery.cssNumber</code> as a list of all CSS properties you might use without a unit. It's used by <a href="/css/"><code>.css()</code></a> to determine if it needs to add <code>px</code> to unitless values.</p>
-    <p>The keys of the <code>jQuery.cssNumber</code> object are camel-cased and the values are all set to <code>true</code>. If you want to prevent the <a href="/css/"><code>.css()</code></a> method from automatically adding the <code>px</code> unit for a specific CSS property, you can add an extra property to the <code>jQuery.cssNumber</code> object.</p>
+    <p>You can think about <code>jQuery.cssNumber</code> as a list of all CSS properties you might use without a unit. Prior to jQuery 4.0, it was used by <a href="/css/"><code>.css()</code></a> to determine if it needs to add <code>px</code> to unitless values.</p>
+    <p>The keys of the <code>jQuery.cssNumber</code> object are camel-cased and the values are all set to <code>true</code>. If you want to prevent the <a href="/css/"><code>.css()</code></a> method from automatically adding the <code>px</code> unit for a specific CSS property and that property is not yet a key of the <code>jQuery.cssNumber</code> object, you can add such an extra property:</p>
     <pre><code>
 jQuery.cssNumber.someCSSProp = true;
     </code></pre>
-    <p>By default the object contains the following properties:</p>
-    <ul>
-      <li><code>zIndex</code></li>
-      <li><code>fontWeight</code></li>
-      <li><code>opacity</code></li>
-      <li><code>zoom</code></li>
-      <li><code>lineHeight</code></li>
-      <li><code>widows</code> (added in jQuery 1.6)</li>
-      <li><code>orphans</code> (added in jQuery 1.6)</li>
-      <li><code>fillOpacity</code> (added in jQuery 1.6.2)</li>
-      <li><code>columnCount</code> (added in jQuery 1.9)</li>
-      <li><code>order</code> (added in jQuery 1.10.2)</li>
-      <li><code>flexGrow</code> (added in jQuery 1.11.1)</li>
-      <li><code>flexShrink</code> (added in jQuery 1.11.1)</li>
```

Comment:
I removed it, but we could consider leaving the list in and updating the new values we added. But it's easy to just check what you have in your jQuery version and if you maintain a library, you shouldn't rely on this list anyway and just pass strings.